FirebaseInstallations/Source/Library/FIRInstallations.m

```diff
@@ -191,7 +225,7 @@ + (void)assertCompatibleIIDVersion {
   return;
 #else
   if (![self isIIDVersionCompatible]) {
-    [NSException raise:NSInternalInconsistencyException
+    [NSException raise:kFirebaseInstallationsErrorDomain
```

Comment:
I am kind of lost with this message and I have no idea what to do here. I have downloaded the info.plist again,  I have checked your other PR, I have checked the release notes, I have also look into the code and still no idea, so this is kind of my last resort 😅.

Reply:
@tiagoalmeida sounds like you are having issue with the Firebase update. I am sorry to hear if it is so. Would you like to create a ticket with a description of your issue so we can track all related details there and potentially help other people having similar problems?